src/models.rs

```diff
@@ -283,28 +290,32 @@ impl OpenAIModels {
         //This is the max tokens allowed between prompt & response
         match self {
             OpenAIModels::Gpt3_5Turbo => RateLimit {
-                tpm: 90_000,
-                rpm: 3_500,
+                tpm: 2_000_000,
+                rpm: 10_000,
             },
             OpenAIModels::Gpt3_5Turbo0613 => RateLimit {
-                tpm: 90_000,
-                rpm: 3_500,
+                tpm: 2_000_000,
+                rpm: 10_000,
             },
             OpenAIModels::Gpt3_5Turbo16k => RateLimit {
-                tpm: 180_000,
-                rpm: 3_500,
+                tpm: 2_000_000,
+                rpm: 10_000,
             },
             OpenAIModels::Gpt4 => RateLimit {
-                tpm: 10_000,
-                rpm: 200,
+                tpm: 300_000,
+                rpm: 10_000,
             },
             OpenAIModels::Gpt4Turbo => RateLimit {
-                tpm: 10_000,
-                rpm: 200,
+                tpm: 2_000_000,
```

Comment:
wow this is a significant bump!!

Reply:
Yeah, I was pretty shocked too, but that's what the documentation now shows